ui/src/app/applications/components/application-details/application-details.tsx

```diff
@@ -36,6 +36,7 @@ interface ApplicationDetailsState {
     slidingPanelPage?: number;
     zoom?: number;
     filteredGraph?: any[];
+    titleDirection?: boolean;
```

Comment:
Perhaps rename this as well since the tooltip is changing.  eg. nameDirection

Or, since it is a boolean, truncateNameOnRight = true for the normal/regular situation.